caravel/assets/spec/javascripts/explorev2/actions_spec.js

```diff
@@ -0,0 +1,16 @@
+import { it, describe } from 'mocha';
+import { expect } from 'chai';
+import * as actions from '../../../javascripts/explorev2/actions/exploreActions';
+import { initialState } from '../../../javascripts/explorev2/stores/store';
+import { exploreReducer } from '../../../javascripts/explorev2/reducers/exploreReducer';
+
+describe('reducers', () => {
+  it('set form data according to given value', () => {
```

Comment:
i wonder if this would be more clear as:
`it('sets correct field value given a key and value'`